src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+]])
+
+-- }}} security configuration
+
+-- {{{ sharding configuration
+
+M['sharding'] = format_text([[
+    This section defines configuration parameters related to sharding.
+]])
+
+M['sharding.bucket_count'] = format_text([[
+    The total number of buckets in a cluster. Learn more in Bucket count.
+]])
+
+M['sharding.connection_outdate_delay'] = format_text([[
+    Time to outdate old objects on reload.
+]])
+
+M['sharding.discovery_mode'] = format_text([[
+    A mode of the background discovery fiber used by the router to find buckets.
+]])
+
+M['sharding.failover_ping_timeout'] = format_text([[
+    The timeout (in seconds) after which a node is considered unavailable if
+    there are no responses during this period. The failover fiber is used to
+    detect if a node is down.
+]])
+
+M['sharding.lock'] = format_text([[
+    Whether a replica set is locked. A locked replica set cannot receive new
+    buckets nor migrate its own buckets.
+]])
+
+M['sharding.rebalancer_disbalance_threshold'] = format_text([[
+    The maximum bucket disbalance threshold (in percent). The disbalance
+    is calculated for each replica set using the following formula:
+
+    `|etalon_bucket_count - real_bucket_count| / etalon_bucket_count * 100`
+]])
+
+M['sharding.rebalancer_max_receiving'] = format_text([[
+    The maximum number of buckets that can be received in parallel by a single
+    replica set. This number must be limited because the rebalancer sends a
+    large number of buckets from the existing replica sets to the newly added
+    one. This produces a heavy load on the new replica set.
+]])
+
+M['sharding.rebalancer_max_sending'] = format_text([[
+    The degree of parallelism for parallel rebalancing.
+]])
+
+M['sharding.rebalancer_mode'] = format_text([[
+    Configure how a rebalancer is selected:
+
+    -`auto` (default): if there are no replica sets with the rebalancer
+    sharding role (`sharding.roles`), a replica set with the rebalancer is
+    selected automatically among all replica sets.
+    - `manual`: one of the replica sets should have the rebalancer sharding
+    role. The rebalancer is in this replica set.
+    - `off`: rebalancing is turned off regardless of whether a replica set
+    with the rebalancer sharding role exists or not.
+]])
+
+M['sharding.roles'] = format_text([[
+    Roles of a replica set in regard to sharding. A replica set can have the
+    following roles:
+
+    - `router`: a replica set acts as a router.
+    - `storage`: a replica set acts as a storage.
+    - `rebalancer`: a replica set acts as a rebalancer.
+
+    The rebalancer role is optional. If it is not specified, a rebalancer is
+    selected automatically from the master instances of replica sets.
+
+    There can be at most one replica set with the rebalancer role.
+    Additionally, this replica set should have a `storage` role.
+]])
+
+M['sharding.roles.*'] = 'Role name.'
```

Comment:
I would just say 'Sharding role: router, storage or rebalancer' here.

Reply:
Okay, let's do it this way.